### PR TITLE
feature/log_output_doc

### PR DESCRIPTION
**Pull request by zhen0427**
pgm-io now provides an interface to configure the log level of the converter, we therefore add explanary information into the documentation.

### CODE REVIEW

File: docs/converters/converter.md [hunk 1]
```diff
@@ -58,3 +58,23 @@ You can also add additional information about each component in the form of `ext
...
 ```
+
+## Configuring The Log Output
+ 
+We have provided an interface to configure the log level of the converter.
+Notice that this log level only belongs to the logger within the converter.
+Users need to set their basic configuration of the `logging` module to a level that is below what is configuired for the converters.
+ 
+```python
+# This is an example in your script
```
Comment (mgovers) [inline docs/converters/converter.md:69]:
```suggestion
# An example of what may be included in your script
```

File: docs/converters/converter.md [hunk 2]
```diff
@@ -58,3 +58,23 @@ You can also add additional information about each component in the form of `ext
...
+import logging
+from power_grid_model_io.converters import VisionExcelConverter
+ 
+logging.basicConfig(level=logging.INFO) # Only levels INFO and above will be logged
+ 
+converter_warning = VisionExcelConverter(input_file, log_level=logging.WARNING) # If there is any logs above WARNING, they will be logged
```
Comment (mgovers) [inline docs/converters/converter.md:75]: it's probably good to mention that this does not affect the `logging.basicConfig` or any other converters created after that. maybe even provide an example of that, e.g.

```suggestion
converter_warning = VisionExcelConverter(input_file, log_level=logging.WARNING) # If there is any logs above WARNING, they will be logged
converter_info = VisionExcelConverter(input_file) # uses default INFO level
```

File: docs/converters/converter.md [hunk 3]
```diff
@@ -58,3 +58,23 @@ You can also add additional information about each component in the form of `ext
...
+ 
+logging.basicConfig(level=logging.WARNING) # Only levels WARNING and above will be logged
```
Comment (mgovers) [inline docs/converters/converter.md:77]: what happens if we change the `basicConfig` after a converter is created before this with default log level settings? e.g. does `converter_info` remain at level `INFO` or does it become `WARNING` at this declaration?

**Comment by mgovers** [comment]
please update the description of the PR

**Comment by Jerry-Jinfeng-Guo** [comment]
> please update the description of the PR

Indeed

File: docs/converters/converter.md (hunk 1 above)
Comment (Jerry-Jinfeng-Guo) [reply docs/converters/converter.md:69]: Reasonable rewording. Or something like:
```suggestion
# Examplary usage in your script
```

File: docs/converters/converter.md (hunk 2 above)
Comment (Jerry-Jinfeng-Guo) [reply docs/converters/converter.md:75]: I think the line below did just that?

File: docs/converters/converter.md (hunk 3 above)
Comment (Jerry-Jinfeng-Guo) [reply docs/converters/converter.md:77]: I agree with this comment and the one above.
```suggestion
logging.basicConfig(level=logging.WARNING) # Only levels WARNING and above will be logged
assert converter_info.get_log_level() == logging.INFO # previously created converters will retain their original log level, regardless of system wide configuration
```

**Review by Jerry-Jinfeng-Guo — APPROVED** [review]

**Pull request merged**